apps/ensadmin/src/components/indexing-status/indexing-stats.tsx

```diff
@@ -289,25 +289,33 @@ export function IndexingStatsForFollowingStatus({
  * UI component for presenting indexing stats UI for specific overall status.
  */
 export function IndexingStatsShell({
-  overallStatus,
+  omnichainStatus,
   children,
-}: PropsWithChildren<{ overallStatus: OverallIndexingStatusId }>) {
+}: PropsWithChildren<{ omnichainStatus: OmnichainIndexingStatusId }>) {
   return (
     <Card className="w-full flex flex-col gap-2">
       <CardHeader>
         <CardTitle className="flex gap-2 items-center">
           <span>Indexing Status</span>
-
+          {/* 
           <Badge
             className={cn(
               "uppercase text-xs leading-none",
-              overallStatus === OverallIndexingStatusIds.IndexerError && "bg-red-600 text-white",
+              omnichainStatus === OmnichainIndexingStatusIds.IndexerError &&
+                "bg-red-600 text-white",
             )}
-            title={`Overall indexing status: ${overallStatus}`}
+            title={`Overall indexing status: ${omnichainStatus}`}
           >
-            {overallStatus === OverallIndexingStatusIds.IndexerError
+            {omnichainStatus === OmnichainIndexingStatusIds.IndexerError
               ? "Indexer Error"
-              : overallStatus}
+              : omnichainStatus}
+          </Badge> */}
+
+          <Badge
+            className={cn("uppercase text-xs leading-none")}
+            title={`Overall indexing status: ${omnichainStatus}`}
+          >
+            {omnichainStatus}
```

Comment:
We don't have `OmnichainIndexingStatusIds.IndexerError` anymore, will need to discuss the way to present the current indexing project unavailable state in the ENSAdmin UI.